ui/imports/shared/views/chat/CompactMessageView.qml

```diff
@@ -162,9 +165,10 @@ Item {
     }
 
     Rectangle {
-        property alias chatText: chatText
-
         id: messageContainer
+
+        property alias messageContent: messageContent
```

Comment:
this line is ok?

Reply:
@borismelnik this is a fix on the qml, not refactor, so that line is just a substitution for the line `property alias chatText: chatText` you may see above. I didn't write that, didn't create that, just updated the current way to point to what it should according to the currently implemented style.

But in general don't worry too much about that, since we have new `Message` component in the `StatusQ` and we are going to introduce it I hope very soon.